MainRepository/Program.cs

```diff
@@ -1,7 +1,13 @@
-﻿internal class Program
-{
-    private static void Main(string[] args)
-    {
-        Console.WriteLine("Hello, World!");
-    }
-}
\ No newline at end of file
+﻿using System.Text;
+// Reduce string
+Console.Write ("Enter the string: ");

```

Comment:
The error can be avoided by using the **!** (eg: Console.ReadLine()!)

![image](https://github.com/karthick-1205/Assignments/assets/16486802/a09b6bcb-5e67-4b8d-8183-23e26c8b1461)